modules/lvgl/Kconfig.input

```diff
@@ -38,4 +42,19 @@ config LV_Z_POINTER_KSCAN_INVERT_Y
 
 endif # LV_Z_POINTER_KSCAN
 
+config LV_Z_POINTER_INPUT
+	bool "Input lvgl pointer"
+	default y
+	depends on INPUT
+	depends on DT_HAS_ZEPHYR_LVGL_POINTER_INPUT_ENABLED
+
+config LV_Z_POINTER_INPUT_MSGQ_COUNT
+	int "Message queue size"
```

Comment:
this (and the button one on the last patch) should be something like "Input pointer queue message count" (better qualified, and less ambigious, size feels like bytes when it's actually messages)